src/app/organizations/organizations/organizations.component.ts

```diff
@@ -53,20 +54,30 @@ export class OrganizationsComponent extends Base implements OnInit {
   ngOnInit() {
     this.isLoggedIn$ = this.trackLoginService.isLoggedIn$;
     this.organizationsStateService.updateSearchNameState('');
-    this.organizationSearchForm = this.formBuilder.group({ name: '' });
+    this.organizationSearchForm = this.formBuilder.group({ name: '', sort: '' });
     this.loading$ = this.alertQuery.showInfo$;
     // The real loading$ is currently not being used because the alertQuery global loading is used instead
     // this.loading$ = this.organizationsQuery.loading$;
     this.organizationsStateService.updateOrganizations();
     this.filteredOrganizations$ = this.organizationsQuery.filteredOrganizations$;
+    this.filteredOrganizations$.pipe(filter(orgs => orgs != null), take(1))
```

Comment:
don't subscribe, just map to another observable and move this to the query.ts

Reply:
@garyluu This length can't be an observable, I only need this once at the beginning

Reply:
the view doesn't do a re-draw if the value is the same value.  if your code only works if on the first observable emit then something is very wrong